core/sugartypes.ml

```diff
@@ -55,7 +58,7 @@ end = struct
 end
 
 (* type variables *)
-type tyvar = Quantifier.t
+(* type tyvar = Quantifier.t *)
```

Comment:
This seems strange, what does the [@@deriving] on the next line refer to?

Reply:
good point. This was supposed to go away, together with   `Sugartypes.tyvar`, `Sugartypes.type_variable`, and  `Sugartypes.quantifier`. Done now.